buildenv/jenkins/JenkinsfileBase

```diff
@@ -440,6 +437,55 @@ def setup() {
 	}
 }
 
+def setup_jck_interactives() {
+	def targetDir = "${env.WORKSPACE}/../../jck_run/jdk${JDK_VERSION}/jdk"
+	if (PLATFORM.contains("windows")) {
+		targetDir = "c:/Users/jenkins/jck_run/jdk${JDK_VERSION}/jdk"
+	}
+	def tarBall = CUSTOMIZED_SDK_URL.tokenize('/').last()
+	echo "tarBall is ${tarBall}"
+	def jckRunDirExists = sh(script: """
+		if [ -d "${targetDir}" ]; then
+			echo true
+		else
+			echo false
+		fi
+		""", returnStdout: true).toBoolean()
+
+	if (jckRunDirExists) {
+		def jdkDir = ""
+		if (PLATFORM.contains("windows")) {
+			jdkDir = sh(script: "unzip -l `pwd`/openjdkbinary/${tarBall} | head -n 4 | tail -n 1 | xargs -n 1 echo | tail -n 1", returnStdout: true).trim().replaceFirst(".\$","")
+			if (PLATFORM.contains("x86-32_windows")) {
+				jdkDir = "${jdkDir}_32"
+			} else if (PLATFORM.contains("x86-64_windows")) {
+				jdkDir = "${jdkDir}_64"
+			}
+		} else {
+			jdkDir =  sh(script: "tar -tf `pwd`/openjdkbinary/${tarBall} | head -1", returnStdout: true).trim().replaceFirst(".\$","")
+		}
+		def jckRunDir = "${targetDir}/${jdkDir}"
+		def jckRunJDKExists = sh(script: """
+		if [ -d "${jckRunDir}" ]; then
+			echo true
+		else
+			echo false
+		fi
+		""", returnStdout: true).toBoolean()
+
+		if (!jckRunJDKExists) {
+			sh returnStatus: true, script: """
+				cd ${targetDir}
+				mkdir ${jdkDir}
+				cd ${jdkDir}
+				cp -r ${TEST_JDK_HOME}/* .
+				chgrp -R jck ${targetDir}/${jdkDir}
+				chmod -R 755 ${targetDir}/${jdkDir}
```

Comment:
This needs to be 775 so that jck users can delete the files

Reply:
Updated.